### PR TITLE
Missing translation key en.spree.activerecord.models.spree/payment.one

### PR DESCRIPTION
**Pull request by romale**
In path: /admin/orders/R445126588/payments/21

image

OFN 3.2.3
Firefox Desktop

#### Release notes

Changelog Category: Added | Changed | Deprecated | Removed | Fixed | Security

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -3635,3 +3635,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
       shipment:
         cannot_ready: "Cannot ready shipment."
       invalid_taxonomy_id: "Invalid taxonomy id."
+    activerecord:
+      models:
+        spree/payment:
```
Comment (sauloperez) [inline config/locales/en.yml:3640]: this key is already defined at line 35 so you should append `one` there.

**Review by sauloperez — CHANGES_REQUESTED** [review]
Thanks again for another useful contribution @romale !

File: config/locales/en.yml (hunk 1 above)
Comment (romale) [reply config/locales/en.yml:3640]: on line 35, 'spree/payment' is under 'activerecord.**attributes**', but in my case under 'activerecord.**models**'. Or does it not matter?

Comment (sauloperez) [reply config/locales/en.yml:3640]: right. Good catch. I find it a bit odd but if it works :+1:

**Review by sauloperez — APPROVED** [review]

**Comment by filipefurtad0** [comment]
Hey @romale ,

I hope to have spotted exactly the bit which now should be translated in production - the page looks good:
image

Looks ready to be merged, moving to ready to go.

Thank you for contributing!

**Comment by romale** [comment]
there will be others, I haven't finished translating the UG yet :))

**Comment by sauloperez** [comment]
My mistake. This should read as "Payment" and not "One".

**Comment by filipefurtad0** [comment]
My mistake as well - I completely missed that...
I see this should be fixed by #5963 . Thanks @sauloperez .

**Pull request merged**